package.json

```diff
@@ -52,7 +54,8 @@
     "typescript": "^3.8.3"
   },
   "peerDependencies": {
-    "eslint": "^6.2.2"
+    "eslint": "^6.2.2",
+    "prettier": "^2.0.4"
```

Comment:
this is probably a breaking change? (if so, needs to be noted in the commit message)

Reply:
Yes, adding prettier requires manually updating existing modules to be compatible, and even though it's easy, it should be documented.

Moving forward, whenever devs make a new one-app module, it should have prettier included, so that change will need to be made to the template.